src/haz3lweb/view/dec/Diag.re

```diff
@@ -18,24 +18,22 @@ let tr_bl =
       (),
     ) =>
   SvgUtil.Path.(
-    {
```

Comment:
This is because reason is updated from 3.11 -> 3.12. I think that's a good thing.